app/controllers/employees_controller.rb

```diff
@@ -11,6 +11,10 @@ def show
 
   def new
     @employee = Employee.new
+    @private_number = rand(111111..999999)
+    if Employee.exists?(private_number: @private_number)
+      @private_number = rand(111111..999999)
```

Comment:
What if the new private number generated has also been already assigned to another Employee?

Reply:
We have validation for that in the model using `uniqueness: true`